Volunteer page

JessSumner opened the pull request:
Why:
The rescue needs to advertise for volunteers.

This PR:
Creates a volunteer page.
Adds text content describing the roles of volunteers.
Adds some rat pictures to brighten up the page.
Has set up a link that will take the user to the Foster Application Form (which is still to be made).

JessSumner committed:
Volunteer page

Generated a controller for volunteer.
Added text content to the volunteer show page.
Added route but the route is raising an error.

JessSumner committed:
Content updated for volunteer page

Content updated and organized.
Formatted using p tags
volunteers.scss file has been created and a few styles have been added to align the text in the center of the page and format the headers of each mini section.
Started to put infrastructure in place for linking to the foster application form.

JessSumner committed:
Images added to the volunteer page

Content has been updated to add side images.
Text has been shifted to the right and block has been made smaller.
Changed file/controller method/route to an index.

JessSumner committed:
Updated image names to follow a sequence.

Why:
I want file names to be informative to anyone who will look at the code. It makes sense to me for all rat images to be called rat followed by an index.

This commit:
Renames rat.jpg to rat-0.jpg
Renames gw.jpg to rat-1.jpg
The files and the pages where the pictures are called have been updated.

JessSumner committed:
Links to forms corrected.

A link has been set up to take the user to the foster application form.
The adoption application form link has been corrected so that the link is within the sentence not after.
The css class name for the main div on the volunteer page has been modified to be volunteer-text-image.

JessSumner committed:
Deleted helper file and test file.

These files are not in use yet so they have been removed.

MattMSumner commented on app/assets/javascripts/volunteers.coffee:
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
How would you feel about removing this file and re-creating if you'd like to add some javascript here?

MattMSumner commented:
1 small comment, otherwise this LGTM.

JessSumner committed:
Volunteer.coffee file removed

File not currently being used amd so it has been deleted to keep the app small.
